app/javascript/components/public_health/PublicHealthHeader.js

```diff
@@ -127,7 +127,7 @@ class PublicHealthHeader extends React.Component {
           {this.state.importType === 'saf' && (
             <div className="mb-3">
               <a href="https://github.com/SaraAlert/SaraAlert/blob/master/public/Sara%20Alert%20Import%20Format.xlsx?raw=true">Download formatting guidance</a>{' '}
-              (Updated 2/9/2021)
+              (Updated 2/24/2021)
```

Comment:
This date should be 3/9/2021 (when 1.24.0 is scheduled to be deployed). Also, when saving excel files, excel remembers which tab and cell you were last on, so we would like to have this file saved with the current tab being the first (Template) and the current cells being the first (A1)

Reply:
done

Reply:
still seeing the default cell for the Guidance tab at 'Contact of Known Case?'

Reply:
really? I just did a fresh pull and I seeing it on the first sheet